src/executor/create_executor.cpp

```diff
@@ -83,6 +86,69 @@ bool CreateExecutor::DExecute() {
                 ResultTypeToString(current_txn->GetResult()).c_str());
     }
   }
+
+  // Check if query was for creating trigger
+  if (node.GetCreateType() == CreateType::TRIGGER) {
+    LOG_INFO("enter CreateType::TRIGGER");
+    std::string database_name = node.GetDatabaseName();
+    std::string table_name = node.GetTableName();
+    std::string trigger_name = node.GetTriggerName();
+    // std::unique_ptr<catalog::Schema> schema(node.GetSchema());
+
+    commands::Trigger newTrigger(node);
+
+    // // new approach: add trigger to the data_table instance directly
+    // storage::DataTable *target_table =
```

Comment:
Actually the approach mentioned in the comment was not right. Please check our correct approach in our latest codes. Useless comments have been removed.